tests/unit/commands/test_borgmatic.py

```diff
@@ -249,41 +249,41 @@ def test_run_configuration_repos_ordered():
     flexmock(module).should_receive('log_error_records').with_args(
         'bar: Error running actions for repository', OSError
     ).and_return(expected_results[1:]).ordered()
-    config = {'location': {'repositories': ['foo', 'bar']}}
+    config = {'location': {'repositories': [{'path': 'foo'}, {'path': 'bar'}]}}
     arguments = {'global': flexmock(monitoring_verbosity=1, dry_run=False), 'create': flexmock()}
     results = list(module.run_configuration('test.yaml', config, arguments))
     assert results == expected_results
 
 
-def test_run_configuration_retries_round_robbin():
```

Comment:
I can't get this one test to pass, something to do with the order these functions are called in:
```bash
flexmock.CallOrderError:
log_error_records("bar: Error running actions for repository", , levelno=30, log_command_error_output=True) 
called before
log_error_records(levelno=30, log_command_error_output=True, message="foo: Error running actions for repository", error=<class 'OSError'>)
```

Any suggestion on what would be the right way to fix this?

Reply:
> Any suggestion on what would be the right way to fix this?

I didn't write this test, but I think what's going on is the config setup is incorrect:

Simply change this:

```python
config = {'location': {'repositories': [{'path':'foo','path':'bar'}]}, 'storage': {'retries': 1}}
```

... to this:

```python
config = {'location': {'repositories': [{'path':'foo'}, {'path':'bar'}]}, 'storage': {'retries': 1}}
```

> (Also this should probably be round_robin)

Yes! Good catch.